docs/source/compile/inductor_profiling.rst

```diff
@@ -1,3 +1,5 @@
+.. _torchinductor-gpu-profiling:
```

Comment:
This is a better fix (I guess this is how you are supposed to control how links show up ? ) but it was also fixed here: https://github.com/pytorch/pytorch/pull/103014/

Reply:
actually nvm, this just so you could link it, not so it would show up different in side panel